assembl/tests/views_tests/test_graphql.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+import json
+
+import pytest
+from graphql_relay.node.node import to_global_id
+
+from assembl import models
+from assembl.graphql.schema import Schema as schema
+
+
+@pytest.fixture(scope="function")
+def graphql_request(request, test_adminuser_webrequest, discussion):
+    req = test_adminuser_webrequest
+    req.matchdict = {"discussion_id": discussion.id}
+    return req
+
+
+def test_get_thematics_noresult(graphql_request):
+    res = schema.execute('query { thematics(identifier:"survey") { id, title, description, numPosts, numContributors, questions { title }, video {title, description, htmlCode} } }', context_value=graphql_request)
+    assert json.loads(json.dumps(res.data)) == {u'thematics': []}
+
+
+def test_get_thematics_no_video(discussion, graphql_request, test_session):
+    title = u"Comprendre les dynamiques et les enjeux"
+    title = models.LangString.create(title, locale_code="fr")
+    thematic = models.Thematic(
```

Comment:
Actually it's not the same code, one is a thematic without video, and the other one with a video, I can't factor it.